cmd/storagenode/gracefulexit.go

```diff
@@ -75,26 +80,26 @@ func cmdGracefulExitInit(cmd *cobra.Command, args []string) error {
 	// get list of satellites
 	satelliteList, err := client.getNonExitingSatellites(ctx)
 	if err != nil {
-		fmt.Println("Can't find any non-existing satellites.")
+		fmt.Println("Can't find any non-exiting satellites.")
```

Comment:
Avoid negation for message. Something like `"Can't find active satellites."` or similar.